windows/src/unit-tests/package-info/test/test.xml

```diff
@@ -0,0 +1,2 @@
opyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\fantuwua.ttf</Name><Description>Font Ethiopic Fantuwua</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\FreeSerif.otf</Name><Description>Font Free Serif</Description><CopyLocation>0</CopyLocation><FileType>.otf</FileType></File><File><Name>..\..\..\shared\fonts\geez\goffer.ttf</Name><Description>Font Ethiopic Yigezu Bisrat Goffer</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\hiwua.ttf</Name><Description>Font Ethiopic Hiwua</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\jiret.ttf</Name><Description>Font Ethiopia Jiret</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\jiretsl.ttf</Name><Description>Font Ethiopia Jiret Slant</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\tint.ttf</Name><Description>Font Ethiopic Tint</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\washrab.ttf</Name><Description>Font Ethiopic WashRa Bold</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\washrabsl.ttf</Name><Description>Font Ethiopic WashRa Bold Slant</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\washrasb.ttf</Name><Description>Font Ethiopic WashRa SemiBold</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\washrasbsl.ttf</Name><Description>Font Ethiopic WashRa SemiBold Slant</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\wookianos.ttf</Name><Description>Font Ethiopic Wookianos</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\shared\fonts\geez\yebse.ttf</Name><Description>Font Ethiopic Yebse</Description><CopyLocation>0</CopyLocation><FileType>.ttf</FileType></File><File><Name>..\..\..\gff\gff_amh_7\build\gff_amh_7.kvk</Name><Description>File gff_amh_7.kvk</Description><CopyLocation>0</CopyLocation><FileType>.kvk</FileType></File><File><Name>..\..\..\gff\gff_amh_7\build\gff_amh_7.kmx</Name><Description>Keyboard Amharic</Description><CopyLocation>0</CopyLocation><FileType>.kmx</FileType></File><File><Name>..\..\..\gff\gff_ethiopic_7\build\gff_ethiopic_7.kvk</Name><Description>File gff_ethiopic_7.kvk</Description><CopyLocation>0</CopyLocation><FileType>.kvk</FileType></File><File><Name>..\..\..\gff\gff_ethiopic_7\build\gff_ethiopic_7.kmx</Name><Description>Keyboard Ethiopic - (Language Neutral)</Description><CopyLocation>0</CopyLocation><FileType>.kmx</FileType></File></Files><Strings/></Package>
```

Comment:
Can you beautify this xml file so it'll be easier to maintain the unit tests and easily see diffs in future PR's?

Reply:
Sure